shared/src/common/interfaces/user/user.interface.ts

```diff
@@ -5,7 +5,7 @@ type Geoposition = {
   long: number
 };
 
-interface IUser {
+type IUser = {
```

Comment:
```suggestion
interface IUser {
```

Reply:
If to use interface then i receive an error `Index signature is missing in type`

Reply:
But it isn't a reason to change it to type 🙂
You need to figurate out what exactly happen. You can ping me in the slack 🙂